common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -238,7 +240,9 @@ protected void before() {
                 protected void after() {
                     LOGGER.info("Deleting connect cluster '{}' after test per @ConnectCluster annotation on {}", cluster.name(), name(element));
                     // delete cm
-                    kubeClient().deleteContent(yaml);
+                    KubernetesClient client = new DefaultKubernetesClient();
+                    client.configMaps().withName(cluster.name()).delete();
```

Comment:
Why was this change necessary? I don't think it works in practice, because although the kubeClient() is correctly namespaced (i.e. according to the `@Namespace`), the DefaultKubernetesClient() isn't. We probably want to provide access to a `NamespacedKubernetesClient` from the `clusterResource`.

Reply:
@tombentley Good catch! I debugged this code and looks like it works now because the necessary namespace is a first in an array of two existing namespaces. So I fixed this bug by using config for `DefaultKubernetesClient` with namespase which can be obtained from Strimzi `KubeClient`.